package-lock.json

```
@@ -19,6 +19,7 @@
         "level": "8.0.1",
         "lodash": "4.17.21",
         "long": "5.2.3",
+        "queue-microtask": "^1.2.3",
```

chore: We should use a fixed version to match the pattern we use on the other dependencies.
```suggestion
        "queue-microtask": "1.2.3",
```

---

Done